src/containers/Cluster/Cluster.tsx

```diff
@@ -164,7 +164,7 @@ function Cluster({
                 />
             </div>
 
-            <div className={b('content')}>{renderTab()}</div>
+            <div>{renderTab()}</div>
```

Comment:
Do we really need this empty div?

Reply:
Yes. I didn't figured out why this happens, but without this `div` some empty space created and table content overlap header
![Screen Shot 2023-07-13 at 12 35 45](https://github.com/ydb-platform/ydb-embedded-ui/assets/67755036/716b21b0-df56-484b-b7e0-b14299052193)